minecraft_servers/chillecke/manifest.json

```diff
@@ -0,0 +1,34 @@
+{
+  "server_name": "chillecke",
+  "nice_name": "ChillEcke",
+  "direct_ip": "chillecke.xyz",
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "web": "https://web.chillecke.xyz",
+    "web_shop": "https://shop.chillecke.xyz",
+    "discord": "https://discord.chillecke.xyz",
```

Comment:
Custom links are reserved for partners only; change to a normal discord.gg/[CODE] link